test/plugin_functional/test_suites/core_plugins/application_status.ts

```diff
@@ -66,6 +66,13 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
   };
 
   describe('application status management', () => {
+    before(async () => {
+      await PageObjects.common.navigateToApp('settings');
+      await PageObjects.settings.clickKibanaSettings();
+      await PageObjects.settings.setAdvancedSettingsSelect('pageNavigation', 'individual');
+      await browser.refresh();
```

Comment:
NIT: if we are going to use this on most suites, should probably be extracted to it's own method on `settings` page object